app/src/main/java/io/intrepid/contest/screens/contestcreation/editcategoriestocontest/EditCategoriesFragment.java

```diff
@@ -80,9 +87,20 @@ public void showCategoriesList() {
         activity.showCategoryList();
     }
 
+    @Override
+    public void showCategory(String previousCategoryName, String previousCategoryDescription) {
```

Comment:
this is really to show the previous Category text. As in when I go to the screen to edit a category, I should first see the pre-existing name in the Edittext before I do any editing. I think there's a better name for this